src/Libraries/CoreNodes/Color.cs

```diff
@@ -479,30 +476,17 @@ public Color GetColorAtParameter(double parameter = 0.0)
         }
     }
 
-    [IsVisibleInDynamoLibrary(false)]
+    [IsVisibleInDynamoLibrary(true)]
     public class ColorRange2D
     {
         private Quadtree quadtree;
-
+        
         private IList<Color.IndexedColor2D> indexedColors;
 
         private ColorRange2D(IEnumerable<Color> colors, IEnumerable<UV> parameters)
         {
             var parameterList = parameters as UV[] ?? parameters.ToArray();
             var colorList = colors as Color[] ?? colors.ToArray();
-
-            quadtree = Quadtree.ByUVs(parameterList);
```

Comment:
Quadtree search was removed. Although faster to find closest neighbors, this caused weird blocky behavior.